### PR TITLE
Add link to Media Session article

### PR DESCRIPTION
**Pull request by beaufortfrancois**
@samdutton Now that the Media Session article is up, I thought we should link it from your existing article at https://developers.google.com/web/updates/2015/07/media-notifications

WDYT?

FYI @mounirlamouri

### CODE REVIEW

File: src/content/en/updates/2015/07/media-notifications.md [hunk 1]
```diff
@@ -41,17 +42,12 @@ The screencast below shows the process of displaying a notification and controll
...
-
-We welcome your feedback on the MediaSession API. You can follow the <a href="https://groups.google.com/a/chromium.org/forum/#!topic/blink-dev/dLWDxYgxzQ8" title="Intent to Implement thread on Google Groups">Intent to Implement discussion</a>, comment on the <a href="https://mediasession.spec.whatwg.org/" title="WHATWG Media Session spec">WHATWG standard</a>, or track our <a href="https://crbug.com/497735" title="Implementation bug on crbug.com">implementation bug</a>.
-
-
-
-
-
+With the [Media Session API](/web/updates/2017/02/media-session), you can
+customize media notifications by providing metadata for the media your web app
+is playing. It also allows you to handle media related events such as seeking
```
Comment (samdutton) [inline src/content/en/updates/2015/07/media-notifications.md]: Nit: 'It' => 'The API'

**Comment by samdutton** [comment]
Thank you! LGTM apart from one tiny suggestion.

**Comment by beaufortfrancois** [comment]
@samdutton Fixed!

**Comment by samdutton** [comment]
👍

**Pull request merged**